src/E1_coldflow/config.h

```diff
@@ -46,7 +46,8 @@ struct valveInfo *valves;
 
 #define HIGH_SOL_PIN 6
 
-#define LOX_ADAPTER_HEATER_PIN 7
+#define LOX_ADAPTER_PT_HEATER_PIN 7
+#define LOX_GEMS_HEATER_PIN 8
```

Comment:
I'm also not sure what this will be. depends on which output you use. 
I think the plan was use the 24V for the gems, and chute for the pt?
in which case `LOX_GEMS_HEATER_PIN 7` and idk what the other one would be. @andycate